packages/commonwealth/client/vite.config.ts

```diff
@@ -17,7 +17,7 @@ function createScriptsResolver(folder: string): Alias {
 
 // https://vitejs.dev/config/
 export default defineConfig(({ mode }) => {
-  const envPath = path.dirname(path.dirname(process.cwd())); // root project .env
+  const envPath = path.dirname(path.dirname(projectRootDir)); // root project .env
```

Comment:
This doesn't haven't the same effect on my system. With `process.cwd()` the `envPath` is the root folder but with `projectRootDir` the `envPath` is `.../packages/`

Reply:
Shouldn't we use the file path here anyway? This is probably a more reliable way to find the root in the repo structure than a running process which could be any plugin or extension using this config.

Reply:
`process.cwd()` and `projectRootDir` are both paths but `projcectRootDir` seems to result in the wrong `envPath`

Reply:
ok.. probably missing another level of path.dirname() to get all the way to the root